packages/remix-dev/cli/run.ts

```diff
@@ -305,7 +305,8 @@ export async function run(argv: string[] = process.argv.slice(2)) {
           templateType! || (answers.appType === "stack" ? "repo" : "template"),
         projectDir,
         remixVersion: flags.remixVersion,
-        installDeps: flags.install ?? answers.install,
+        installDeps: flags.install || answers.install,
+        useTypeScript: flags.typescript || answers.useTypeScript,
         useTypeScript: flags.typescript ?? answers.useTypeScript,
```

Comment:
heh oops, the commit suggestion didn't let me remove the previous `useTypeScript` 🤦